src/main/kotlin/world/anhgelus/msmp/alteredrespawn/DefaultCondition.kt

```diff
@@ -0,0 +1,18 @@
+package world.anhgelus.msmp.alteredrespawn
+
+import org.bukkit.GameMode
+import org.bukkit.entity.Player
+
+class DefaultCondition {
```

Comment:
What is the goal of this class?

Reply:
The goal of this class is to create functions to initialize default condition of player 

I admit it can be changed just to setCondition and add : - Default
- When a player died

Reply:
I think we can refactor it to simplify it.